pytest-embedded-idf/pytest_embedded_idf/serial.py

```diff
@@ -194,6 +194,36 @@ def dump_flash(
         else:
             return content
 
+    @EspSerial.use_esptool
+    def erase_partition(self, partition_name: str = None) -> None:
```

Comment:
how about no default value here?

```suggestion
    def erase_partition(self, partition_name: str) -> None:
```